packages/rich-text/src/helpers/editor.spec.ts

```diff
@@ -0,0 +1,269 @@
+        {
+          type: BLOCKS.TABLE_ROW,
+          data: {},
+          isVoid: false,
+          children: [
+            {
+              type: BLOCKS.TABLE_HEADER_CELL,
+              data: {},
+              isVoid: false,
+              children: [paragraph('header 1')],
+            },
+            {
+              type: BLOCKS.TABLE_HEADER_CELL,
+              data: {},
+              isVoid: false,
+              children: [paragraph('header 2')],
+            },
+          ],
+        },
+        {
+          type: BLOCKS.TABLE_ROW,
+          data: {},
+          isVoid: false,
+          children: [
+            {
+              type: BLOCKS.TABLE_CELL,
+              data: {},
+              isVoid: false,
+              children: [paragraph('cell 1')],
+            },
+            {
+              type: BLOCKS.TABLE_CELL,
+              data: {},
+              isVoid: false,
+              children: [paragraph('cell 2')],
+            },
+          ],
+        },
+      ],
+    };
+    const editor = createEditor([table]);
+    const entry: NodeEntry = [table, [0]];
+
+    expect(slateNodeEntryToText(editor, entry)).toEqual([
+      paragraph('header 1'),
+      paragraph('header 2'),
+      paragraph('cell 1'),
+      paragraph('cell 2'),
+    ]);
+  });
+
+  it('blockquote', () => {
+    const blockquote: CustomElement = {
+      data: {},
+      isVoid: false,
+      type: BLOCKS.QUOTE,
+      children: [paragraph('text 1'), paragraph('text 2')],
+    };
+    const editor = createEditor([blockquote]);
+    const entry: NodeEntry = [blockquote, [0]];
+
+    expect(slateNodeEntryToText(editor, entry)).toEqual([paragraph('text 1'), paragraph('text 2')]);
+  });
+
+  it('list - UL', () => {
+    const ul: CustomElement = {
+      data: {},
+      isVoid: false,
+      type: BLOCKS.UL_LIST,
+      children: [
+        {
+          data: {},
+          isVoid: false,
+          type: BLOCKS.LIST_ITEM,
+          children: [
+            {
+              type: BLOCKS.UL_LIST,
+              isVoid: false,
+              data: {},
+              children: [
+                {
+                  data: {},
+                  isVoid: false,
+                  type: BLOCKS.LIST_ITEM,
+                  children: [paragraph('text 1')],
+                },
+              ],
+            },
+          ],
+        },
+        {
+          data: {},
+          isVoid: false,
+          type: BLOCKS.LIST_ITEM,
+          children: [paragraph('text 2')],
+        },
+        {
+          data: {},
+          isVoid: false,
+          type: BLOCKS.LIST_ITEM,
+          children: [paragraph('text 3')],
+        },
+      ],
+    };
+    const editor = createEditor([ul]);
+    const entry: NodeEntry = [ul, [0]];
+
+    expect(slateNodeEntryToText(editor, entry)).toEqual([
+      paragraph('text 1'),
+      paragraph('text 2'),
+      paragraph('text 3'),
+    ]);
+  });
+
+  it('list - OL', () => {
+    const ol: CustomElement = {
```

Comment:
Almost seems like an exact copy of the one above.
Why not create a function that returns your mock like

```ts
const newList(listType: BLOCKS.UL_LIST|BLOCKS.OL_LIST) => ({
  data: {},
  isVoid: false,
  type: listType,
  children: [
    { ...
```

or perhaps even just combine the two cases and test with a nested list that contains both types.

Reply:
Sure :)